### DISCUSSION TITLE
Issue in Importing ODF data through import wizard

### DISCUSSION THREAD

**Discussion by Ashwin-Kumar-Godasu** [mtex-toolbox/mtex#1458] [Ask Anything]
Hello everyone,

I am facing an error when importing the ODF data, error is as follows
**Could not detect file format. You may ask for help in the MTEX forum (https://github.com/mtex-toolbox/mtex/discussions).**

I have tried to export the data in both generic ASCII File and MTEX ASCII File, and exporting was successful. 
 But I am not able to import ODF from IMPORT WIZARD.

please help me

**Comment by kilir** [comment]
Hi,
see here:
https://github.com/mtex-toolbox/mtex/discussions/1431
Cheers,
Rüdiger

**Reply by Ashwin-Kumar-Godasu** [reply]
Thank you sir, It works. I also want to know how to impose specimen symmetry on the imported ODF and plot it. For example, I want to impose orthorhombic specimen symmetry.

**Comment by lfgmorales** [comment]
you may want to have a look at


https://mtex-toolbox.github.io/DetectionOfSampleSymmetry.html


cheers


Luiz


Luiz F. G. Morales
ScopeM & Structural Geology and Tectonics Group
ETH Zürich
HPM C57.3

Otto-Stern-Weg 3

8093 Zürich

Switzerland

Phone: + 41 44 633 3746

<tel:+41446333746>E-mail: ***@***.***



"No trees were destroyed in the sending of this message, however, a significant number of electrons were terribly inconvenienced"


________________________________

**Reply by Ashwin-Kumar-Godasu** [reply]
thank you for the help sir
